### PR TITLE
Re-word README.md

### PR DESCRIPTION
**Pull request by meowchanicyuumi**
I fixed some (no offense) badly translated stuff in the README so that it was a bit more legible.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +47,6 @@ Parts of the lyrics editor in this ruleset were inspired by them.
 
 - [Appveyor](https://www.appveyor.com/), [CodeFactor](https://www.codefactor.io/) and [Github action](https://github.com/features/actions) for providing free `CI`/`CD` service.
 
-- [Figma](https://www.figma.com/), for fast creating assets like logo or icon.
+- [Figma](https://www.figma.com/), for fast creation of assets like logos or icon.
 
-- [Miro](https://miro.com/). Used to declare the complex question by drawing the graph.
+- [Miro](https://miro.com/). Used for project management and planning.
```
Comment (andy840119) [inline README.md]: Miro is a tool for drawing something like flow chart, not for project management.

File: README.md [hunk 2]
```diff
@@ -16,8 +16,7 @@ The source code of the `karaoke` ruleset, running on [osu!lazer](https://github.
 
 Some features are still under heavy development, but existing features should work fine.
 
-And notice that this project not have much [demo](https://github.com/karaoke-dev/sample-beatmap) can try, and most of them are written in Japanese.
-So recommend looking this project around for seeing new feature instead of actually using it.
+Please note that this project doesn't have much of a [demo](https://github.com/karaoke-dev/sample-beatmap) currently available, and most of the lyrics are written in Japanese, so we recommend looking around this project to find new features instead of actually using it, unless you understand / speak Japanese.
```
Comment (andy840119) [inline README.md:19]: Is `doesn't have much of a demo` not have grammar issue? I did not seem this kinds of grammar before 🤔 
.
Also, another reason for looking around is that this project is not stable, i hope user start using it until most of the bugs are fixed, and the editor has been implemented.

**Review by andy840119 — COMMENTED** [review]
Thank you for your effort 🎉

File: README.md (hunk 1 above)
Comment (andy840119) [reply README.md]: And it used to help me decide the architecture for some features.

Comment (borealis17) [reply README.md]: I updated that miro line just a few seconds ago

File: README.md (hunk 2 above)
Comment (borealis17) [reply README.md:19]: `doesn't have much of a demo` functions somewhat similarly to (sorry for my bad japanese, i'm still learning) `これは示威運動をありません` or saying "has an unfinished demo".

sorry if that's confusing, i'm bad at explaining things

Comment (andy840119) [reply README.md:19]: There are some of the demos in [this repo](https://github.com/karaoke-dev/sample-beatmap), so i'm not really sure what does that "a" means 🤔 
.
> Also, another reason for looking around is that this project is not stable, i hope user start using it until most of the bugs are fixed, and the editor has been implemented.

Also, could you add something like this into this PR? I forgot to add this before :P

**Comment by meowchanicyuumi** [comment]
oops! i accidentally closed the pr, so i just reopened it.

File: README.md (hunk 2 above)
Comment (borealis17) [reply README.md:19]: yeah! i'll add/push that

**Comment by andy840119** [comment]
Guess it's ok enough to make the merge 🙆🏼‍♂️

**Pull request merged**